presto-jdbc/src/main/java/com/facebook/presto/jdbc/HttpClientCreator.java

```diff
@@ -33,20 +35,31 @@
     private final String userAgent;
     private final JettyIoPool jettyIoPool;
     private final Map<String, BiConsumer<HttpClientConfig, String>> configSetters;
+    private final String user;
+    private final Optional<String> password;
 
     public HttpClientCreator(
             String userAgent,
             JettyIoPool jettyIoPool,
-            Map<String, BiConsumer<HttpClientConfig, String>> configSetters)
+            Map<String, BiConsumer<HttpClientConfig, String>> configSetters,
+            String user,
```

Comment:
`userAgent` - is the name of the client app the Presto server is being queries (e.g. `presto-cli`, `presto-jdbc`, etc.). `user` - is the login user.